### DISCUSSION TITLE
How to set debug log level to bare minimum or disable logs completely ?

### DISCUSSION THREAD

**Discussion by geminigeek** [browserless/chrome#4191] [Q&A]
hi,
i have a question regarding logs in docker container. how can i set debug level so that i will have only error and critical logs, currently i have too many logs ? 

the logs i am having issue with are  like these check below, it seems they are not api logs they are logs of chromium browser! can anyone help ? 

i am using browserless docker image the latest version

```
browserless.io:ChromeCDPWebSocketRoute:trace 192.168.1.11 Setting up file:// protocol request rejection +0ms
browserless.io:ChromeCDPWebSocketRoute:trace 192.168.1.11 Navigation to about:blank +31ms

...
 browserless.io:ChromeCDPWebSocketRoute:trace 192.168.1.11 GET: https://<redacted>
 browserless.io:ChromeCDPWebSocketRoute:trace 192.168.1.11 GET: https://<redacted>
 browserless.io:ChromeCDPWebSocketRoute:trace 192.168.1.11 GET: https://<redacted>
 browserless.io:ChromeCDPWebSocketRoute:trace 192.168.1.11 GET: https://<redacted>


```

**Comment by geminigeek** [comment]
i found it myself , i had to recreate docker compose to make it work 

```
- DEBUG=-*
```

**Reply by mjsommer** [reply]
@geminigeek , where and how did you use `- DEBUG=-*` to stop [DEBUG] logging?

When I use Browserless in a container, my console if filled with: `[1746831656.754][DEBUG]: DevTools WebSocket Event: Log.entryAdded .................`

Thanks,
Martin

**Reply by geminigeek** [reply]
set this as environment variable in docker compose

**Reply by mjsommer** [reply]
Right, my `docker-compose.yml` looks like this ...

```
version: '3.8'
services:
  browserless:
    image: browserless/chrome:latest
    ports:
      - "3000:3000"
    environment:
      - "DEBUG=-*"

```

... and, I am getting this ...

`[1746899855.796][DEBUG]: DevTools WebSocket Command: Page.getFrameTree (id=292) (session_id=76E1 ...`

What am I missing?

Martin

**Reply by geminigeek** [reply]
try removing "" around debug , and run like this 

```
docker compose up -d --force-recreate --remove-orphans
```

**Reply by mjsommer** [reply]
Thanks, but no change. I wonder if the [DEBUG] lines are forced, because they are associated with an error?

```
browserless-1  | [1746904540.935][DEBUG]: DevTools WebSocket Event: Log.entryAdded (session_id=D96B77734A893645998182A09820949E) 99E13FA8FDFD5BE25515C2C34D6DD338 {
browserless-1  |    "entry": {
browserless-1  |       "level": "error",
browserless-1  |       "lineNumber": 8664,
browserless-1  |       "source": "security",
browserless-1  |       "stackTrace": {
browserless-1  |          "callFrames": [ {
browserless-1  |             "columnNumber": 7,

```

Ref: https://github.com/browserless/browserless/discussions/4723

I am running Browerless and my own app, in separate containers, and spinning them up from the same docker-compose.yml file. They see one another, but getting continuous "security" errors, while trying to communicate.

**Reply by mjsommer** [reply]
@geminigeek , any more insights here?

**Reply by geminigeek** [reply]
Just try create a standalone docker compose with just one container, see if the logs works as intended

**Reply by mjsommer** [reply]
Yes, everything is clean running "alone". The issue, is when I attempt to communicate with it, from my app.
